modules/common/wikiquote.js

```diff
@@ -7,135 +7,139 @@ var request = require.safe('request');
+    */
+    exports.getQuotesForSection = function(pageId, sectionIndex, success) {
+        var url = API_URL + '?format=json&action=parse&noimages=&pageid=' + pageId + '&section=' + sectionIndex;
+        request.get(url, function(failurer, response, body) {
+            var result = JSON.parse(body),
+                quotes = result.parse.text['*'],
+                quoteArray = [],
+                regex = /\<li\>((.|\n)*?)\<\/.*\n/g,
+                match = quotes.match(regex);
 
-         do {
-          match = regex.exec(quotes);
-          if (match) {
-            var quote = match[1].trim(),
-                tagRegex = /\<(.|\n)*?\>|\<.*?\/\>/g;
-            quote = quote.replace(tagRegex, '');
-            //Hack to try and stop getting actors names
-            if (quote.length < 25) continue;
-            quoteArray[quoteArray.length] = quote;
-          }
-         } while (match);
+            do {
+                match = regex.exec(quotes);
+                if (match) {
+                    var quote = match[1].trim(),
+                        tagRegex = /\<(.|\n)*?\>|\<.*?\/\>/g;
+                    quote = quote.replace(tagRegex, '');
+                    // Hack to try and stop getting actors names
+                    if (quote.length < 25)  {
+                        continue;
+                    }
+                    quoteArray[quoteArray.length] = quote;
+                }
+            } while (match);
 
-        success({ titles: result.parse.title, quotes: quoteArray });
-      });
-  };
+            success({ titles: result.parse.title, quotes: quoteArray });
+        });
+    };
 
-  /**
-   * Get a random quote for the given title search.
-   * This function searches for a page id for the given title, chooses a random
-   * section from the list of sections for the page, and then chooses a random
-   * quote from that section.  Returns the titles that were used in case there
-   * is a redirect.
-   */
-  exports.getRandomQuote = function(titles, success, error) {
+    /**
+    * Get a random quote for the given title search.
+    * This function searches for a page id for the given title, chooses a random
+    * section from the list of sections for the page, and then chooses a random
+    * quote from that section.  Returns the titles that were used in case there
+    * is a redirect.
+    */
+    exports.getRandomQuote = function(titles, success, error) {
 
     var errorFunction = function(msg) {
       error(msg);
     };
 
     var chooseQuote = function(quotes) {
-      var randomNum = Math.floor(Math.random()*quotes.quotes.length);
+      var randomNum = Math.floor(Math.random() * quotes.quotes.length);
       success({ titles: quotes.titles, quote: quotes.quotes[randomNum] });
     };
 
     var getQuotes = function(pageId, sections) {
-      var randomNum = Math.floor(Math.random()*sections.sections.length);
+      var randomNum = Math.floor(Math.random() * sections.sections.length);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 space characters but found 6.](https://www.codacy.com/app/theprojectshop/Kassy/file/2651361737/issues/source?bid=3282987&fileBranchId=3324570#l142)